pkg/pillar/cmd/zedagent/handlenetworkinstance.go

```diff
@@ -879,3 +879,10 @@ func timeNanoToProto(timenum int64) *timestamp.Timestamp {
 func writeSentFlowProtoMessage(contents []byte) {
 	writeProtoMessage("lastflowlog", contents)
 }
+
+func handleAppContainerMetricsModify(ctxArg interface{}, key string,
+	statusArg interface{}) {
+
+	acMetrics := statusArg.(types.AppContainerMetrics)
+	log.Infof("handleAppContainerMetricsModify(%s), num containers %d", key, len(acMetrics.StatsList))
```

Comment:
This will result in log messages when nothing changed in the config or status. Better to make log.Debug for that reason.

Reply:
ok.